mobile/cmp/button/Button.scss

```diff
@@ -1,10 +1,171 @@
+      background-color: var(--xh-button-bg);
+      .xh-button--enabled {
+        &:active,
+        &.xh-button--active {
+          color: var(--xh-button-active-text-color);
+          background-color: var(--xh-button-active-bg);
+        }
+      }
+    }
+
+    &.xh-button--intent-primary {
+      color: white;
+      background-color: var(--xh-intent-primary);
+      .xh-button--enabled {
+        &:active,
+        &.xh-button--active {
+          background-color: var(--xh-intent-primary-darkest);
+        }
+      }
+    }
+
+    &.xh-button--intent-success {
+      color: white;
+      background-color: var(--xh-intent-success);
+      .xh-button--enabled {
+        &:active,
+        &.xh-button--active {
+          background-color: var(--xh-intent-success-darkest);
+        }
+      }
+    }
+
+    &.xh-button--intent-warning {
+      color: white;
+      background-color: var(--xh-intent-warning);
+      .xh-button--enabled {
+        &:active,
+        &.xh-button--active {
+          background-color: var(--xh-intent-warning-darkest);
+        }
+      }
+    }
+
+    &.xh-button--intent-danger {
+      color: white;
+      background-color: var(--xh-intent-danger);
+      .xh-button--enabled {
+        &:active,
+        &.xh-button--active {
+          background-color: var(--xh-intent-danger-darkest);
+        }
+      }
+    }
+  }
+
+  // Minimal mode.
+  &--minimal,
+  &--minimal.xh-button--enabled:active {
+    background-color: transparent;
+
+    &.xh-button--intent-none {
+      color: var(--xh-text-color);
+      &.xh-button--enabled.xh-button--active {
+        background-color: var(--xh-bg-alt);
+      }
+    }
+
+    &.xh-button--intent-primary {
+      color: var(--xh-intent-primary-darker);
+      &.xh-button--enabled.xh-button--active {
+        background-color: var(--xh-intent-primary-trans2);
+      }
+    }
+
+    &.xh-button--intent-success {
+      color: var(--xh-intent-success-darker);
+      &.xh-button--enabled.xh-button--active {
+        background-color: var(--xh-intent-success-trans2);
+      }
+    }
+
+    &.xh-button--intent-warning {
+      color: var(--xh-intent-warning-darker);
+      &.xh-button--enabled.xh-button--active {
+        background-color: var(--xh-intent-warning-trans2);
+      }
+    }
+
+    &.xh-button--intent-danger {
+      color: var(--xh-intent-danger-darker);
+      &.xh-button--enabled.xh-button--active {
+        background-color: var(--xh-intent-danger-trans2);
+      }
+    }
+  }
+
+  // Outlined mode
+  &--outlined,
+  &--outlined.xh-button--enabled:active {
+    background-color: transparent;
+    border: 1px solid var(--xh-button-bg);
+
+    &.xh-button--intent-none {
+      color: var(--xh-button-bg);
+      border-color: var(--xh-button-bg);
+      &.xh-button--enabled.xh-button--active {
+        background-color: var(--xh-button-bg);
+      }
+    }
+
+    &.xh-button--intent-primary {
+      color: var(--xh-intent-primary-darker);
```

Comment:
This looked a bit off when in a toolbar, especially when in dark mode, where the darker color appears to be more muted relative to the borders. Looks like outlined desktop buttons w/intents use the same color for text + border

Reply:
On desktop, the text color is the `-darker` variant of the border's intent color like here - but only for the light theme. I'll adjust mobile to use the `-lighter` variant for both text and border in dark theme, which will get us full parity with desktop.